api/nakadi-event-bus-api.yaml

```diff
@@ -723,39 +736,138 @@ definitions:
   Event:
     type: object
     description: |
-      **To be changed**. This is the most general representation of an event, that can be processed
-      by Nakadi.
+      **Note** The Event definition will be externalized in future versions of this document.
+
+      A basic payload of an Event. The actual schema is dependent on the information configured for the
```

Comment:
Distinguish between e.g. payload and metadata, payload and control data, payload and headers. Otherwise, talking about payload here, confuses me. Sure, the entire event is the payload of e.g. a Kafka Event, but that's probably not what we are talking about.